.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ env:
 
 jobs:
   build:
-    runs-on: 'ubuntu-20.04'
+    runs-on: ubuntu-latest
```

Comment:
i think we should run on latest by default. if needed - we can introduce running on older versions in another PR